cmd_move.go

```diff
@@ -23,7 +23,7 @@ func commandMove(
 		log.Fatal(tre.New(err, "get source key failed", "key", sourceKey))
 	}
 	// store value for key to target
-	commandPutPasteGenerate(kmsService, storageService, target, "put", targetKey, sourceValue)
+	commandPutPasteGenerate(kmsService, storageService, target, "put", targetKey, sourceValue, true)
```

Comment:
tip:  create a constant with value true and use that constant here. Better readible